smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -661,10 +665,63 @@ def locationHandler(evt) {
 
 def doDeviceSync(){
 	log.trace "Doing Hue Device Sync!"
+
+	// Check if state.updating failed to clear
+	if (state.lastUpdateStarted < (now() - 20 * 1000) && state.updating) {
+		state.updating = false
+		log.warn "state.updating failed to clear"
+	}
+
 	convertBulbListToMap()
 	poll()
 	ssdpSubscribe()
 	discoverBridges()
```

Comment:
I used to use this to monitor if the IP changed. Now that we have local polling. Don't we have a better way to do this?